config/ftbquests/quests/chapters/pneumaticcraft_repressurized.snbt

```diff
@@ -707,7 +709,7 @@
 			x: -5.0d
 			y: -7.0d
 			subtitle: "What a Corny Joke"
-			description: [""]
+			description: ["Isn't that great of a fuel on its own, but it can be refined!"]
```

Comment:
This doesn't really make sense to me. 
`A great fuel on its own, but it can be refined!` is a bit better.

Reply:
If I recall correctly I did took that sentence from the PNC manual 😅

Reply:
More exactly there is a note about Ethanol being a bad fuel on its own, but it can be used to produce Biodiesel which is good :D

Reply:
`Ethanol could be burned on its own, but it could be refined, what is definitely worth doing!` Is much better I think :D

Reply:
Agreed!